### ISSUE TITLE
ACC-270.16(Level A) [fdp-table],  icon-tab-bar [Platform] , [fd-checkbox] in  fd-list  is getting activated without releasing the spacebar

### ISSUE THREAD

**Issue by Meggie886** [SAP/fundamental-ngx#13237]
Is it a bug ?

Yes 

From all of the versions 

1. In the table, if there is a group bar,it is triggered without releasing the respective key (for example pressing the space keyboard
https://stackblitz.com/run?file=src%2Fapp%2Fplatform-table-initial-state-example.component.html

![Image](https://github.com/user-attachments/assets/a83a5280-31cc-4546-bba9-21e048a0ebcc)

2. The row navigation in table[Platform].  Event triggered before keyborad releasing when press spacebar
https://stackblitz.com/run?file=src%2Fapp%2Fplatform-table-navigatable-row-indicator-example.component.ts

3. Content changes before keyborad releasing when press spacebar
https://stackblitz.com/run?file=src%2Fapp%2Fexample%2Ficon-tab-bar-async-tabs-example.component.html

4. The checkbox in fd-list is actived before releasing the spacebar 
https://stackblitz.com/run?file=src%2Fapp%2Flist-selection-example.component.html

If you cannot open the example code link, just open the example from the document and try. They are all examples from the document.

Expect: Only activated by releasing the spacebar

Test ACC-270.16: Is functionality being operated with a single pointer action (tap or click) not triggered by the down-event, if the down-event is not essential (like typing text)?
Follow the test case
Make use of touch gestures and mouse clicks.
Check where interactive UI elements are presented (content, which can be clicked or tapped, or triggered via keyboard).
Check all buttons. Remark: Data input in input elements happens on the down event. Do not test input fields.
Check for other elements which trigger functions. Focus & Selections can be set with a 'down' event.
Check the input devices
Touch:
tap the element, then keep the finger on the touch screen.
check if something happens. If something is triggered while the finger is not lifted: fail.
Lift the finger. Check if the function is now triggered: passed.
tap again. Move finger out of the visual estate of the element. lift finger. Check if the function is not executed: passed.
Mouse
click the element, keep the mouse button down.
check if something happens. If something is triggered while the mouse button is held down: fail
release the mouse button. Check if the function is now triggered. passed.
click again. Move mouse pointer out of the visual estate of the element. release button. Check if the function is not executed: passed.
Keyboard: On buttons the SPACE key without modifiers is the only key to be checked for this requirement.
move focus onto a button element
use SPACE bar, press the SPACE bar and keep it pressed.
check if a function is triggered while holding down the SPACE bar: fail.
release the SPACE bar. Check if the function is now executed: passed.
press key again and additionally press ESCAPE or SHIFT. Release the keys (Shift/Space, in any sequence). Check if the function is not executed: passed.
Realized
(1) Functions are not executed on down events.
Partially Realized
(1) Some functions are executed on down events: <list of functions>
Not Realized
(1) Functions are executed on down events.

<a id="acc27017"></a>

**Comment by droshev** [comment]
@Meggie886 the stackblitz example links are not saved and they don't work. Meanwhile i checked the platform [table](https://fundamental-ngx.netlify.app/#/platform/table/basic) "Initial State" and the groupings are not acting on spacebar pressing at all.

**Comment by droshev** [comment]
@nikolay-kolarov Can you advise what to do. UI5 is behaving the same way https://ui5.sap.com/#/entity/sap.m.List/sample/sap.m.sample.ListSelection

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

I gave guidance via mail. Please let me know in case further support from my side is needed.

Thanks,
Nikolay

**Comment by droshev** [comment]
@Meggie886 according to the discussion with @nikolay-kolarov who had discussed that issue with the central a11y team there is no A11Y issue with that. I am closing the issue. If you have questions, feel free to reach out the central accessibility team

**Comment by Meggie886** [comment]
Hi @droshev This is an accessibility issue from I confirmed from the ntral accessibility team.

**Comment by droshev** [comment]
@Meggie886 Can you share more info from the accessibility team?

**Comment by Meggie886** [comment]
I made a record of fundamental example of standard list with checkbox. And tried use mouse at first, then press the enter key without release, the event triggers all the time which is not right. It's behaviour should be trigger the event when pressup. @droshev 

https://github.com/user-attachments/assets/7dfa926b-520c-41e1-b388-aff75669f076

> [@Meggie886](https://github.com/Meggie886) Can you share more info from the accessibility team?

**Comment by droshev** [comment]
@Meggie886 Can you share what the accessibility team reported? what requirement was not achieved?

**Comment by Meggie886** [comment]
This is from ACC document 
<img width="1067" height="291" alt="Image" src="https://github.com/user-attachments/assets/67b3e201-db81-4855-a189-5b9ca72d9159" />
And this is from jira ticket the QA expected 

<img width="720" height="97" alt="Image" src="https://github.com/user-attachments/assets/a8900744-f646-4176-9c30-dc65a706a553" />

**Comment by droshev** [comment]
@nikolay-kolarov Can you verify whether this is applicable here?

**Comment by droshev** [comment]
The input from the accessibility is that we should not trigger this select/unselect if the spacebar is only down. the change should happen on key out. UI5 would need to change that too. more info in Test `ACC-270.16`
